lxd/instance_state.go

```diff
@@ -199,8 +200,8 @@ func instanceStatePut(d *Daemon, r *http.Request) response.Response {
 		return doInstanceStatePut(inst, req)
 	}
 
-	resources := map[string][]string{}
-	resources["instances"] = []string{name}
+	resources := map[string][]api.URL{}
+	resources["instances"] = []api.URL{*api.NewURL().Path(version.APIVersion, "instances", name, "state")}
```

Comment:
This should still be a URL to the instance itself, not its state endpoint, as that is not a resource per-se.